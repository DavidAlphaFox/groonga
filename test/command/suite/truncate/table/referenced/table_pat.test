table_create Tags TABLE_PAT_KEY ShortText

table_create TagReferences TABLE_HASH_KEY Tags
load --table TagReferences
[
{"_key": "Great"}
]

select TagReferences

truncate Tags

select TagReferences
