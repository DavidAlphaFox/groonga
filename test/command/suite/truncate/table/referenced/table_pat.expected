table_create Tags TABLE_PAT_KEY ShortText
[[0,0.0,0.0],true]
table_create TagReferences TABLE_HASH_KEY Tags
[[0,0.0,0.0],true]
load --table TagReferences
[
{"_key": "Great"}
]
[[0,0.0,0.0],1]
select TagReferences
[[0,0.0,0.0],[[[1],[["_id","UInt32"],["_key","ShortText"]],[1,"Great"]]]]
truncate Tags
[[0,0.0,0.0],true]
select TagReferences
[[0,0.0,0.0],[[[0],[["_id","UInt32"],["_key","ShortText"]]]]]
